### ISSUE TITLE
Bug in Assets page for Azure clusters

### ISSUE THREAD

**Issue by thomasvn** [kubecost/cost-analyzer-helm-chart#2386]
**Describe the bug**  
The Assets page for Azure AKS clusters does not show “Network” costs when Cloud Billing Integration is enabled. This likely has to do with reconciliation. No issues with AWS and GCP.

**To Reproduce**  

1. Go to `/assets` view
2. Click on “Edit Report”
3. Add Filter “Provider: Azure”
4. Notice that there are no line items for “Network”

**Expected behavior**  
Users should see a “Network” line item. Then when drilling in, they should see a Network line item associated with each VMSS (nodegroup).

**Screenshots**  
This is filtered for “kc-demo-dev” which is an Azure Cluster

<img width="1434" alt="Screenshot 2023-06-23 at 10 20 37 AM" src="https://github.com/kubecost/cost-analyzer-helm-chart/assets/11251627/ccd2c972-023f-42af-9e23-c96b8dc602de">

**What impact will this have on your ability to get value out of Kubecost?**
Costs are accurate for the nodes. But not fully accurate when rolling up to the cluster level due to missing Network costs.

**Comment by thomasvn** [comment]
Additionally, I believe this bug was initially addressed in v1.102 with PR: https://github.com/kubecost/kubecost-cost-model/pull/1342

However, this issue is still persisting and I am consistently getting this error:

```
ERR Azure: failed to parse vmss number for VM %!s(<nil>): did not find number in provided string
```

**Comment by kwombach12** [comment]
Thanks @thomasvn I have logged this and we will take a look in 105
